tests/tablespaces/test_tablespaces_04.py

```diff
@@ -0,0 +1,14 @@
+from nose.tools import *
+from tests.tablespaces.tablespace_utils import node2
+
+
```

Comment:
Same as before, the pause/unpause approach needs some comments as what to expect and why from the test file. I'm not sure that this time we can stop the pg_autoctl process without stopping the whole container, so we might just have to accept the control-from-outside-the-test-file idea there.